accre_jupyter/form.yml

```diff
@@ -43,20 +44,25 @@ attributes:
           data-hide-bc-num-cores: true,
           data-hide-cpuarch: true,
           data-hide-qos: true,
+          data-hide-gpus-type-interactive: true,
           data-set-cpuarch: "any"
         ]
       - [ 'interactive (standard nodes, low-latency)', 'interactive',
           data-hide-gpus-type: true,
+          data-hide-gpus-type-interactive: true,
           data-hide-numgpus: true
         ]
+      - [ 'interactive_gpu (GPU accelerated nodes, high-throughput)', 'interactive_gpu',
+          data-hide-gpus-type: true
```

Comment:
I would recommend that we also hide the cpuarch selector and set it to "any" for interactive_gpu for now, but if you see a good reason to keep it then please disregard this suggestion.